geomstats/information_geometry/statistical_metric.py

```diff
@@ -422,3 +423,50 @@ def amari_divergence_tensor(self, base_point):
         first_primal = self.primal_connection.first_kind_christoffels(base_point)
         first_dual = self.dual_connection.first_kind_christoffels(base_point)
         return first_dual - first_primal
+
+class PotentialFunction:
+    def __init__(self, potential_function):
+        self.potential_function = potential_function
+        self.hessian_function = self.hessian()
+        self.third_derivative_function = self.third_derivative()
+
+    def __call__(self, x):
+        return self.potential_function(x)
+
+    def hessian(self):
```

Comment:
receive `x` and return result, not function. (Imagine we make an non autodiff-based implementation of the hessian)